bin/craft

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Craft console bootstrap file
+ */
+
+if (!empty($_composer_autoload_path)) {
+    $basePath = dirname($_composer_autoload_path) . '/../';
+}
+else {
+    $basePath = getcwd();
+}
+
+// Define path constants
+define('CRAFT_BASE_PATH', $basePath);
+define('CRAFT_VENDOR_PATH', CRAFT_BASE_PATH . '/vendor');
```

Comment:
This assumes [`vendor-dir`](https://getcomposer.org/doc/06-config.md#vendor-dir) wasn’t edited at all.